### PR TITLE
Migrations generation mention for the EVM tutorial

### PR DESCRIPTION
**Pull request by abernatskiy**
Migrations generation now has to be done by hand. Mentioning that in the EVM tutorial.

	modified:   docs/tutorials/create-an-evm-processing-squid.md

### CODE REVIEW

File: docs/tutorials/create-an-evm-processing-squid.md [hunk 1]
```diff
@@ -93,10 +93,11 @@ It's worth noting a couple of things in this [schema definition](https://docs.su
 
 The template already has automatically generated TypeScript classes for this schema definition. They can be found under `src/model/generated`.
 
-Whenever changes are made to the schema, new TypeScript entity classes have to be generated, and to do that you'll have to run the `codegen` tool:
+Whenever changes are made to the schema, new TypeScript entity classes have to be generated and migrations have to be updated. To do that run
 
 ```bash
 make codegen
+make migration
```
Comment (dzhelezov) [inline docs/tutorials/create-an-evm-processing-squid.md:100]: should it be:
```
rm -rf db/migrations/*.js
make down
make up
npm run build
make migration
```

**Review by dzhelezov — CHANGES_REQUESTED** [review]

**Comment by abernatskiy** [comment]
> Yes, for the case when a change is introduced into an already existing schema. With a new squid `make codegen; make migration suffices`. Let me restructure that fragment a bit and come back.
Upon a closer investigation - no, the two commands do not suffice.

**Comment by dzhelezov** [comment]
Staking migrations proved to be a bad practice -- any meaningful change will likely require a full resync anyway. So it's almost always a good idea to start from a clean db. The only exception is adding an index

**Comment by abernatskiy** [comment]
The idea was to update migrations more easily before the first `make up`. But upon a closer look at the behavior of `squid-typeorm-migration generate` I realized that the old migrations would have to be manually removed anyway, so I opted to just give the instructions for a general schema change. See the new PR.

**Pull request closed without merging**